tests/scripts/x86_ci_script.sh

```diff
@@ -37,13 +37,13 @@ __buildDirName="$__buildOS.$__buildArch.$__buildConfig"
 set -x
 set -e
 
-__dockerImage="hseok82/dotnet-buildtools-prereqs:ubuntu1604_cross_prereqs_v3_x86"
+__dockerImage="hseok82/dotnet-buildtools-prereqs:ubuntu-16.04-crossx86-ef0ac75-20175511035548"
```

Comment:
@hseok-oh Can you work with @MichaelSimons to get the Docker file created for Linux X86 so that Michael can create the Docker image which can then be consumed from the same location as other images?